configurable_lti_consumer/configurable_lti_consumer.py

```diff
@@ -1,10 +1,18 @@
-from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 
 from lti_consumer import LtiConsumerXBlock
 
 from .exceptions import ConfigurableLTIConsumerException
 from .defaults import CONFIGURABLE_LTI_CONSUMER_SETTINGS
 
+# Prevent Django settings circular import: if an ImproperlyConfigured exception
+# is raised, it means that we are loading Django settings and not instanciating
+# the Xblock (hopefully).
+try:
+    from .defaults import CONFIGURABLE_LTI_CONSUMER_SETTINGS
+except ImproperlyConfigured:
+    pass
+
```

Comment:
@jmaupetit @rmoch I'm thinking maybe it's simpler to on set the `CONFIGURABLE_LTI_CONSUMER_SETTINGS` variable in defaults.py and place the defaulting getter in your code... Especially since you're only using it once...

so in defaults.py:
```python
CONFIGURABLE_LTI_CONSUMER_SETTINGS = {}
```

and in configurable_lti_consumer/configurable_lti_consumer.py:
```python
from defaults.py import CONFIGURABLE_LTI_CONSUMER_SETTINGS

@classmethod
def get_configuration(cls, name):
    for conf_name, configuration in getattr(settings, "CONFIGURABLE_LTI_CONSUMER_SETTINGS", CONFIGURABLE_LTI_CONSUMER_SETTINGS).items():
    ...
```

Reply:
I've implemented your suggestion, thanks @sampaccoud !